ts/src/test/static/response/ellipx.json

```diff
@@ -2024,6 +2024,24 @@
                     "stopLossPrice": null
                 }
             }
+        ],
+        "fetchTicker": [
```

Comment:
you're putting static/request tests in the static/response folder, hence the error